xbmc/pvr/PVRPlaybackState.cpp

```diff
@@ -374,8 +374,7 @@ void CPVRPlaybackState::StartPlayback(CFileItem* item,
         const std::shared_ptr<CPVREpgInfoTag> epgTag = item->GetPVRChannelInfoTag()->GetEPGNow();
         if (epgTag)
         {
-          delete item;
-          item = new CFileItem(epgTag);
+          item.reset(new CFileItem(epgTag));
```

Comment:
I'm more in favor for `item = std::make_unique<CFileItem>(epgTag);`, like you do at the other places.

Reply:
Done